src/toolkit/pyroprocessing/pyre_volox.cc

```diff
@@ -0,0 +1,60 @@
+#include "pyre_volox.h"
+
+using cyclus::Material;
+using cyclus::Composition;
+using cyclus::CompMap;
+
+namespace pyro {
+
+Volox::Volox() {
+  SetCoeff();
+  temp(0);
+  Rtime(0);
+  flowrate(0);
+  volume(0);
+}
+
+Volox::Volox(double new_temp, 
+             double new_Rtime, 
+             double new_flowrate, 
+             double new_volume
+        ) 
+        {
+          SetCoeff();
+          temp(new_temp);
+          Rtime(new_Rtime);
+          flowrate(new_flowrate);
+          volume(new_volume); 
+}
+
+void Volox::SetCoeff() {
+  th.push_back(4.369E-9);
+  th.push_back(-1.0833E-5);
+  th.push_back(0.008069);
+  th.push_back(-0.9726);
+  ti.push_back(0.2903);
+  ti.push_back(-1.696);
+  r.push_back(0.12435);
+  r.push_back(0.7985);
+}
```

Comment:
same here where those magic number come from ?
to what do they corresponds ?

Reply:
Something bother me with the way you set those coefficient... would it be possible to have default value set in the header, where the reference of them is ?
something like:
`vector<double> th = {4.369E-9, ... , 0.2903};`